_usage/locales.md

```diff
@@ -19,109 +19,117 @@ encoding. Babelfish will automatically translate from the server
 encoding to the client encoding depending on the collation of each
 output column.
 
-Babelfish uses version 153.80 of the ICU collation library. For
-detailed information about PostgreSQL collation behavior, visit [the
-PostgreSQL website](https://www.postgresql.org/docs/14/collation.html).
+For detailed information about PostgreSQL collation behavior, visit [the
+PostgreSQL website](https://www.postgresql.org/docs/13/collation.html).
 
 Babelfish supports *deterministic* and *non-deterministic*
 collations:
 
 - A collation that is *deterministic* will consider two characters as
-  equal if, and only if, they have the same byte sequence; therefore x
-  is not equal to X for a deterministic collation. Collations that are
+  equal if, and only if, they have the same byte sequence.  For example, `x`
+  is not equal to `X` for a deterministic collation.  Collations that are
   deterministic are case-sensitive (CS) and accent-sensitive (AS).
 
-- A non-deterministic collation does not require an identical match;
-  case-insensitivity (CI) is an example of a non-deterministic
-  collation. To have x compare equally to X, choose a
+- A non-deterministic collation does not require an identical match.
+  Case-insensitivity (CI) is an example of a non-deterministic
+  collation. To have `x` compare equal to `X`, choose a
   non-deterministic collation that supports case-insensitivity.
 
 Babelfish and SQL Server follow a naming convention for collations
 that describe the collation attributes, as shown in the table below:
 
 | Attribute   | Description                                            |
 | ----------- | ------------------------------------------------------ |
-| AI          | Accent insensitive                                     |
-| AS          | Accent sensitive                                       |
-| BIN         | BIN requests data to be sorted in code point order, treating the first character as a wchar. Code point order is a fast deterministic collation. |
-| BIN2        | BIN2 requests data to be sorted in code point order. Code point order is a fast deterministic collation. |
-| CI          | Case insensitive                                       |
-| CS          | Case sensitive                                         |
-| PREF        | To sort uppercase letters before lowercase letters, use a PREF collation. If comparison is case-insensitive, the uppercase version of a letter sorts before the lowercase version, if there is no other distinction. The ICU library supports uppercase preference with `colCaseFirst=upper`, but not for `CI_AS` collations. PREF can be applied only to `CS_AS` - deterministic.  |
-
-PostgreSQL doesn\'t support the `LIKE` clause on non-deterministic
-collations, but Babelfish supports it for `CI_AS` collations.
-Babelfish doesn\'t support `LIKE` on AI collations. Pattern matching
+| `ai`        | Accent insensitive                                     |
```

Comment:
it's an arbitrary choice to use AI vs ai, etc., but in SQL Server the collation names are generally written with upper case.  My preference would be to keep it in upper case.